src/Lesson2/Lesson2.java

```diff
@@ -0,0 +1,224 @@
+                "Стул - стілець", "Штопор - штопор", "Коробка переключения передач - коробка перемикання передач",
+                "Змей Горыныч - змій Горинич"};
+        /*Массив со списком високосных годов для задания 5*/
+        int[] year = {1764, 1768, 1772, 1776, 1780, 1784, 1788, 1792, 1796, 1804, 1808, 1812, 1816, 1820,
+                1824, 1828, 1832, 1836, 1840, 1844, 1848, 1852, 1856, 1860, 1864, 1868, 1872, 1876, 1880,
+                1884, 1888, 1892, 1896, 1904, 1908, 1912, 1916, 1920, 1924, 1928, 1932, 1936, 1940, 1944,
+                1948, 1952, 1956, 1960, 1964, 1968, 1972, 1976, 1980, 1984, 1988, 1992, 1996, 2000, 2004,
+                2008, 2012, 2016, 2020, 2024, 2028, 2032,2036, 2040, 2044, 2048, 2052, 2056, 2060, 2064,
+                2068, 2072, 2080, 2084, 2088, 2092, 2096, 2104, 2108, 2112, 2116, 2120, 2124, 2128, 2132,
+                400, 800, 1200, 1600, 2000, 2400};
+
+        Random random = new Random();
+        // Здесь формирую меню выбора задания
+        for (;;) {
+            menuAll: {
+                char choice;
+                do {//это текст меню
+                    System.out.println("Справка: введите число соот. номеру задания\n\n" +
+                    "1: Задание 1. Написать метод, принимающий на вход два целых числа\n" +
+                            "и проверяющий, что их сумма лежит в пределах от 10 до 20 (включительно),\n" +
+                            "если да – вернуть true, в противном случае – false.\n\n" +
+                    "2: Задание 2. Написать метод, которому в качестве" +
+                            " параметра передается целое число, метод должен\n" +
+                            "напечатать в консоль, положительное ли число передали или отрицательное. Замечание: ноль\n" +
+                            "считаем положительным числом.\n\n" +
+                    "3: Задание 3. Написать метод, которому в качестве параметра " +
+                            "передается целое число. Метод должен\n" +
+                            "вернуть true, если число отрицательное, и вернуть false если положительное.\n\n" +
+                    "4: Задание 4. Написать метод, которому в качестве аргументов " +
+                            "передается строка и число, метод должен\n" +
+                            "отпечатать в консоль указанную строку, указанное количество раз.\n\n" +
+                    "5: Задание 5. Написать метод, который определяет, " +
+                            "является ли год високосным, и возвращает boolean\n" +
+                            "(високосный - true, не високосный - false). Каждый 4-й год " +
+                            "является високосным, кроме каждого\n" +
+                            "100-го, при этом каждый 400-й – високосный\n\n" +
+                    "6: Рисование *\n" +
+                    "q: Завершить программу");
+
+                    choice = (char) System.in.read();
```

Comment:
К вопросу о чтении char с помощью Scanner
```suggestion
                    // Declare the object and initialize with
        // predefined standard input object
        Scanner sc = new Scanner(System.in);
   
        // Character input
        choice = sc.next().charAt(0);
```